handler/handler.go

```diff
@@ -0,0 +1,121 @@
+package handler
+
+import (
```

Comment:
This section is still a bit higgledy-piggledy (looks like just this file though).

Reply:
Fixed!